nettacker/core/app.py

```
@@ -289,7 +289,67 @@ def scan_target(
+        canon_by_index = [None] * len(targets)
+
+        def _task(idx, t):
+            ok, canon = resolve_quick(t, timeout_sec=timeout_per_target)
+            return idx, t, (canon if ok and canon else None)
+
+        with ThreadPoolExecutor(max_workers=max_workers) as ex:
+            futures = [ex.submit(_task, i, t) for i, t in enumerate(targets)]
+            for fut in as_completed(futures):
+                try:
+                    idx, orig_target, canon = fut.result()
+                except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)